generic/tests/cfg/dd_test.cfg

```diff
@@ -5,8 +5,9 @@
     images += " stg1"
     create_image_stg1 = yes
```

Comment:
Suggest to use 'force_create_image_stg1 = yes', since case will  boot up guest with the existed images which maybe created partitions.